### PR TITLE
feat: add flatbuffers A6 module for Python.

### PR DESCRIPTION
**Pull request by shuaijinchao**
This module is built using flatc(version: 2.0.0)

### CODE REVIEW

**Review by spacewander — APPROVED** [review]

**Review by tzssangglass — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -5,3 +5,14 @@ make compiled
 ```
 
 Compile to generated files.
+
+## Node
```
Comment (spacewander) [inline README.md]:
```suggestion
## Requirement
```

File: README.md [hunk 2]
```diff
@@ -5,3 +5,14 @@ make compiled
...
+
+The warehouse code is automatically generated using `flatc`, the following is the version information.
```
Comment (spacewander) [inline README.md]: Should be "require to use flatc 2.0.0 to generate the code blah"

**Review by spacewander — APPROVED** [review]

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,2 @@
+.idea
+.vscode
```
Comment (membphis) [inline .gitignore:2]: A blank line is required at the end of the file

File: python/A6/DataEntry.py [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+    """This method is deprecated. Please switch to AddValue."""
+    return AddValue(builder, value)
+def StartValueVector(builder, numElems): return builder.StartVector(1, numElems, 1)
+def DataEntryStartValueVector(builder, numElems):
+    """This method is deprecated. Please switch to Start."""
+    return StartValueVector(builder, numElems)
+def End(builder): return builder.EndObject()
+def DataEntryEnd(builder):
+    """This method is deprecated. Please switch to End."""
+    return End(builder)
```
Comment (membphis) [inline python/A6/DataEntry.py:80]: ditto

File: python/A6/Err/Resp.py [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+    """This method is deprecated. Please switch to Start."""
+    return Start(builder)
+def AddCode(builder, code): builder.PrependUint8Slot(0, code, 0)
+def RespAddCode(builder, code):
+    """This method is deprecated. Please switch to AddCode."""
+    return AddCode(builder, code)
+def End(builder): return builder.EndObject()
+def RespEnd(builder):
+    """This method is deprecated. Please switch to End."""
+    return End(builder)
```
Comment (membphis) [inline python/A6/Err/Resp.py:45]: you'd better update your IDE setting

**Pull request merged**